pkgs/development/libraries/science/chemistry/mmtf-cpp/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
     description = "A library of exchange-correlation functionals with arbitrary-order derivatives";
     homepage = "https://github.com/rcsb/mmtf-cpp";
     license = licenses.mit;
-    platforms = platforms.linux;
+    platforms = platforms.unix;
```

Comment:
should this be `with lib.platforms; linux ++ darwin;` ?

Reply:
We have no reason why it can't work on other BSDs?

Reply:
I am pro BSD support, it's just there is no evidence that it works on BSD's.

Maybe to be more specific, it was obvious to me that we want all platforms but it may or may not work on platforms other than what CI builds. In those cases do we want to guess of the best or list what we know for sure ?

Reply:
On those platforms with lower support tiers, we just normally assume it works. If someone found any problems, they can later fix that. If we don't, the maintainers of these platforms need to add tons of attrs to make them work (which will very likely be rejected by Nixpkgs).

Reply:
thank you, that's was pretty much my question. is it documented anywhere ?

Reply:
Since the package has no platform-specific features and is built with platform-independent tools, I expect it to work well on other UNIX systems.
If it doesn't work, we shouldn't care.

I don't know if it is documented or if there is a common consensus among the committers, but the tier of the systems can be found here.
https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md